crates/sncast/src/response/cast_message.rs

```diff
@@ -0,0 +1,43 @@
+use foundry_ui::{Message, OutputFormat};
+use serde::Serialize;
+use serde_json::Value;
+
+use crate::NumbersFormat;
+
+use super::{
+    command::CommandResponse,
+    print::{Format, OutputData},
+};
+
+#[derive(Serialize)]
+pub struct SncastMessage<T: Serialize> {
+    pub command: String,
+    pub command_response: T,
+    pub numbers_format: NumbersFormat,
+}
```

Comment:
```suggestion
#[derive(Serialize)]
pub struct SncastMessage<T> {
    pub command: String,
    pub command_response: T,
    pub numbers_format: NumbersFormat,
}
```
That Trait bound is required by derive

Reply:
In fact, it should be `CommandResponse` instead of `Serialized`, changed.